lib/tapjoy/autoscaling_bootstrap/AWS/Autoscaling/group.rb

```diff
@@ -9,11 +9,10 @@ def client
               @client ||= Tapjoy::AutoscalingBootstrap::AWS::Autoscaling.client
             end
 
-            def resize(min_size: 0, max_size: 0, desired_capacity:0)
+            def resize(min: 0, max: 0, desired:0)
               self.client.update_auto_scaling_group(
                 auto_scaling_group_name: Tapjoy::AutoscalingBootstrap.scaler_name,
-                min_size: min_size, max_size: max_size,
-                desired_capacity: desired_capacity)
+                min_size: min, max_size: max, desired_capacity: desired)
```

Comment:
Gotcha
On Oct 20, 2015 1:17 PM, "Ali Tayarani" <notifications@github.com> wrote:

> In lib/tapjoy/autoscaling_bootstrap/AWS/Autoscaling/group.rb
> <https://github.com/Tapjoy/tass/pull/15#discussion_r42524758>:
>
> >                self.client.update_auto_scaling_group(
> >                  auto_scaling_group_name: Tapjoy::AutoscalingBootstrap.scaler_name,
> > -                min_size: min_size, max_size: max_size,
> > -                desired_capacity: desired_capacity)
> > +                min_size: min, max_size: max, desired_capacity: desired)
>
> No, since this is a call to AWS and they expect the keys to be like this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Tapjoy/tass/pull/15/files#r42524758>.
>